internal/commands/buildpack_package.go

```diff
@@ -100,6 +104,26 @@ func BuildpackPackage(logger logging.Logger, cfg config.Config, packager Buildpa
 				logger.Warn("Flattening a buildpack package could break the distribution specification. Please use it with caution.")
 			}
 
+			targets, isCompositeBP, err := processBuildpackPackageTargets(flags.Path, packageConfigReader, bpPackageCfg)
+			if err != nil {
+				return err
+			}
+			daemon := !flags.Publish && flags.Format == ""
+			multiArchCfg, err := processMultiArchitectureConfig(logger, flags.Targets, targets, daemon)
+			if err != nil {
+				return err
+			}
+
+			if len(multiArchCfg.Targets()) == 0 {
```

Comment:
```suggestion
			if len(multiArchCfg.Targets()) == 0 && bpPackageCfg.Platform.OS != "" && !isCompositeBP {
```
We probably don't want to warn every time there are no targets, only when the user provided "platform" in the package config? IDK

Reply:
Initially, my idea was to add the warning to get attention to the new feature.  

**Scenario**
Buildpack Authors uses the current configuration (single platform), the update `pack` but they are not aware of the multi-platform effort

Reply:
When they do provide os in package.toml we should probably warn, but other cases should look more like "pro tip: multi-arch support is now available" or something a lil less heavy handed. Just my 2 cents

Reply:
(to be updated in pending commit)